cmd/bootkube/main.go

```
@@ -36,7 +36,6 @@ func main() {
 
 	cmdRoot.AddCommand(cmdVersion)
 	if err := cmdRoot.Execute(); err != nil {
-		fmt.Fprintln(os.Stderr, err)
```

Add a comment explaining why you're ignoring the error?